### PR TITLE
Checks what percent of users you have sent a phone number to

### PR DESCRIPTION
**Pull request by gajeam**
This version checks how many people and what percent of matches you've sent a phone number. It uses [this regular expression from SO](https://stackoverflow.com/a/16699507/1031615) to check for a phone number. It doesn't collect any data on what your number is. I also rounded the percents down to two decimals.

Cases I've tested:
* A message has a phone number in the beginning, middle, or end
* You've sent a person your phone number multiple times (only counts one)
* You've never sent anyone your number

### CODE REVIEW

File: lib/analyze.py [hunk 1]
```diff
@@ -47,22 +53,27 @@
...
+tot_ghost_string = "You've ghosted at least " + str(u_ghosted + they_ghosted) + ", or at least " + str(round((u_ghosted+they_ghosted)/matches * 100, 2)) + "% of all the people you match with\n"
 u_ghost_string = "Total you've dropped the ball: " + str(u_ghosted) + " (you sent a like and they matched, no chatting) \n"
 they_ghost_string = "Total they've dropped the ball " + str(they_ghosted) + " (they sent a like and you matched, no chatting) \n"
 they_dunk_string = "Total shots they've shot and scored: " + str(they_dunk) + " (they sent a like and you matched) \n"
 u_dunk_string = "Total shots you've shot and scored: " + str(u_dunk) + " (you sent a like and they matched) \n"
 tot_likes = "Total shots you've shot: " + str(likes) + " (you sent a like)\n"
 tot_matches = "Total slam dunks: " + str(matches) + " (you've matched with someone or they've matched with you) \n"
 tot_chats = "Total people you've blessed with your time: " + str(chats) + " (you sent a message at least once) \n"
+tot_nubers = "You've sent a phone number to " + str(numbers_sent) + " people on Hinge, or " + str(round((numbers_sent/matches) * 100, 2)) + "% of all the people you match with.\n"
```
Comment (lofi-frog) [inline lib/analyze.py]: nooberz

```suggestion
tot_numbers = "You've sent a phone number to " + str(numbers_sent) + " people on Hinge, or " + str(round((numbers_sent/matches) * 100, 2)) + "% of all the people you match with.\n"
```

File: lib/analyze.py [hunk 2]
```diff
@@ -47,22 +53,27 @@
...
 
 analysis = tot_string + tot_likes + tot_matches + tot_chats
 analysis += "\n------------- \n\n\n"
 analysis += "Fun Stuff ~ \n\n"
-analysis += rej_string + tot_ghost_string + u_ghost_string + they_ghost_string + they_dunk_string + u_dunk_string
+analysis += rej_string + tot_ghost_string + u_ghost_string + they_ghost_string + they_dunk_string + u_dunk_string + tot_nubers
```
Comment (lofi-frog) [inline lib/analyze.py]:
```suggestion
analysis += rej_string + tot_ghost_string + u_ghost_string + they_ghost_string + they_dunk_string + u_dunk_string + tot_numbers
```

**Review by lofi-frog — COMMENTED** [review]
📞🔥

File: lib/analyze.py (hunk 1 above)
Comment (gajeam) [reply lib/analyze.py]: **n00berz

**Review by emam678 — APPROVED** [review]
LGTM 🚢  Thanks for contributing 😄. Feel free to merge at your convenience.

**Comment by gajeam** [comment]
Thanks! @emam97 I don't have write access so I think you have to be the one to merge.

**Pull request merged**